src/ghdcbot/adapters/storage/sqlite.py

```
@@ -950,6 +965,162 @@ def update_social_profile_display(
+                    ORDER BY updated_at DESC
+                    LIMIT 1
+                    """,
+                    (repo, pr_number),
+                ).fetchone()
+
+        if not row:
+            return None
+        data = dict(row)
+        try:
+            data["events"] = json.loads(data.get("events_json") or "[]")
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)